### ISSUE TITLE
Environment recommendations for contributing?

### ISSUE THREAD

**Issue by groovenectar** [estradanic/ksms#3]
Hi there...

Wondering if you have recommendations on software to use for contributing...

Some things I'd love to try:

1) Pull in contacts

2) Create conversation view that will "catch" incoming notifications during a session

But I'm wondering if there's an IDE that you recommend to build / update the UI, compiling etc...

Thanks for any guidance

**Comment by estradanic** [comment]
Hello!

I've actually sort of abandoned this project now, as the kde connect team is working on an official desktop messaging app that will be much better than this one could ever be. [Here's a quick article I found on it.](https://clivejo.com/kde-connect-sms-app/) This github repo is basically just a Qt ui that pipes commands to kde-connect through the command line. It was meant for personal use and it really isn't integrated nicely with kde-connect itself. If you really would like to tinker with it though, I can say that I used Qt Creator for everything except for the AppImage packaging.

If you'd like to contribute to efforts on the official project, here are some relevant links:

- Phabricator: https://phabricator.kde.org/project/profile/159/
- Telegram: https://t.me/joinchat/AOS6gA37orb2dZCLhqbZjg
- Mailing List: https://mail.kde.org/mailman/listinfo/kdeconnect
